build.rs

```diff
@@ -1,25 +1,11 @@
-#[cfg(target_os = "macos")]
 extern crate cc;
 
-#[cfg(target_os = "macos")]
 fn main() {
-    let is_ios = std::env::var("CARGO_CFG_TARGET_OS")
-        .map(|s| s == "ios")
-        .unwrap_or(false);
-
-    if !is_ios {
-        cc::Build::new()
-            .file("src/apple/macos/disk.m")
-            .compile("disk");
-        // DiskArbitration is not available on iOS: https://developer.apple.com/documentation/diskarbitration
-        println!("cargo:rustc-link-lib=framework=DiskArbitration");
-        // IOKit is not available on iOS: https://developer.apple.com/documentation/iokit
+    if std::env::var("TARGET").unwrap().contains("-apple") {
+        cc::Build::new().file("src/mac/disk.m").compile("disk");
         println!("cargo:rustc-link-lib=framework=IOKit");
+        println!("cargo:rustc-link-lib=framework=CoreFoundation");
```

Comment:
Isn't it going to be an issue for iOS since you removed the frameworks condition?

Reply:
It does. The CI job caught part of the breakage [here](https://github.com/GuillaumeGomez/sysinfo/pull/404/checks?check_run_id=1695732460). Please re-add the iOS gated compilation and linking skipping that was here previously.

Reply:
Ah yeah, sorry I didn't look at how much had changed, this now incorporates the ios changes.

Reply:
Thanks 👍